hooks/post_gen_project.py

```diff
@@ -84,6 +85,8 @@ def post_gen():
     subprocess.call(["git", "add", "."])
     subprocess.call(["git", "commit", "-m", "Initial commit"])
 
-
+if '{{cookiecutter.code_of_conduct}}' == 'None':
```

Comment:
could you move this to a new function called `code_of_conduct_clean_up` and call that inside `post_gen` after `move_selected_doc_dir`, please?